### PR TITLE
Make app deployable to gh-pages

### PR DESCRIPTION
**Pull request by zantetsuken88**

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -2,13 +2,15 @@ import React, {Component} from 'react';
...
+const DOMAIN = "/stevenajcox.com";
+
 export default class Navbar extends Component {
   render() {
     return (
       <StyledNavbar>
         <NavbarLinks>
           <NavbarLinksList>
-            <NavbarLink exact to="/">
+            <NavbarLink exact to={`${DOMAIN}/`}>
```
Comment (julesnuggy) [inline src/components/Navbar.jsx]: I think the Footer component will need to have this as well.

File: src/App.jsx [hunk 2]
```diff
@@ -14,6 +14,8 @@ import { faHome, faLightbulb, faHandshake, faAddressCard } from '@fortawesome/fr
 
 library.add(faLinkedin, faTwitter, faInstagram, faHome, faLightbulb, faHandshake, faAddressCard);
 
+const DOMAIN = "/stevenajcox.com";
```
Comment (julesnuggy) [inline src/App.jsx]: As this is used in more than one component, for maintainability, we should put this in a dedicated file under the `styleConstants` directory (which should be renamed `globalConstants` to remain relevant) and then we import it wherever it's needed.

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -2,13 +2,15 @@ import React, {Component} from 'react';
 import { StyledNavbar, NavbarLinks, NavbarLinksList, NavbarLink } from './styles/Navbar';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
+const DOMAIN = "/stevenajcox.com";
```
Comment (julesnuggy) [inline src/components/Navbar.jsx]: See comment above.

**Comment by zantetsuken88** [comment]
Had to rebase and fix merge conflicts which meant force pushing dismissed your approval. I'll override and merge anyway since I didn't change anything and I've test it all works.

**Pull request merged**